### DISCUSSION TITLE
Github Actions Runners -- Custom Labels with ubicloud?

### DISCUSSION THREAD

**Discussion by WilliamAGH** [ubicloud/ubicloud#2480] [Q&A] [answered]
Is there any way to change to custom labels for our Github Actions runners?

e.g., from:
`runs-on: ubicloud-standard-2`

to:
`my-own-syntax`

?

This would help integrate ubicloud into our existing workflows.

Alternatively, is there some method for using any of a group of self hosted runner label options instead of only 1?

**Comment by enescakir** [comment, marked as answer]
Hi @WilliamAGH, Thank you for getting in touch with us!

Right now, we don’t have either feature available.

The second feature isn’t possible with our current architecture. We add "repository-level" self-hosted runners, but runner groups are a feature for "organization-level" self-hosted runners. When we attempt to use a value for `runner_group_id` other than `1` (the default group), it fails.

On the other hand, we can implement a feature for customer labels. Could you explain a bit more about your workflows? How do you plan to use this feature to integrate with your existing processes?

**Comment by aamederen** [comment]
Hi @WilliamAGH,

I just wanted to let you know that I'm closing this discussion for now. As @enescakir asked, if you can let us know a bit more about your workflows and processes, we would be happy to consider it.
